src/main/java/rx/internal/util/PlatformDependent.java

```diff
@@ -20,31 +20,49 @@
 
 /**
  * Allow platform dependent logic such as checks for Android.
- * 
+ *
  * Modeled after Netty with some code copy/pasted from: https://github.com/netty/netty/blob/master/common/src/main/java/io/netty/util/internal/PlatformDependent.java
  */
 public final class PlatformDependent {
 
-    private static final boolean IS_ANDROID = isAndroid0();
+    // Will be null if current platform is not Android
+    // Or if Android API Version can not be resolved.
+    private static final Integer ANDROID_API_VERSION = resolveAndroidApiVersion();
```

Comment:
Is there an API version 0? If not, we could use `int` here so there is no need for unboxing and null checking elsewhere.